src/shared_modules/keystore/qa/test_keystore.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+from subprocess import Popen, PIPE
+import logging
+import os
+import pytest
+import trustme
+from pathlib import Path
+
+LOGGER = logging.getLogger(__name__)
+KEYSTORE_BINARY = "./wazuh-keystore"
+KEYSTORE_TESTTOOL_BINARY = "./wazuh-keystore-testtool"
+CERTS_PATH = 'etc/'
+PRIVATE_KEY_FILE = "etc/sslmanager.key"
```

Comment:
The keypair used for keystore was changed in 4.8.1:
https://github.com/wazuh/wazuh/pull/24357
https://github.com/wazuh/wazuh/issues/24111